config.toml.sample

```diff
@@ -9,7 +9,7 @@ template = "super" # Results template
 rules_json = "/etc/super/rules.json" # Vulnerability rules JSON
 
 # Vulnerable or potentially vulnerable permissions
-[[permissions]]
+[unknown_permissions]
```

Comment:
Yes, makes sense, nevertheless, `name` should be removed, since it's always going to be *unknown*.